bundle-workflow/src/test_workflow/assume_role.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Usage
+#
+# ./assume-role.sh $CLIENT_ROLE_ARN client
+
```

Comment:
To assume role for CLI it should be done using aws sts and this is required to run `cdk deploy` with a different account.
Ref: https://docs.aws.amazon.com/cli/latest/reference/sts/assume-role.html